haskell/README.md

```diff
@@ -5,12 +5,23 @@ SPDX-License-Identifier: MIT
 
 # Stablecoin Haskell
 
-This folder contains a Haskell library to interact with the stablecoin contract (basically Haskell bindings) and tests for it.
+This folder contains:
+
+1. a Haskell library to interact with the stablecoin contract (basically Haskell bindings) and tests for it.
+2. a `stablecoin-client` executable to deploy and interact with the stablecoin contract.
 
 ## Build Instructions
 
 You need [Stack](http://haskellstack.org/) to build this package.
-Run `stack build` to build the library.
+
+Run `stack build` to build the library and the executable.
+Run `stack install` to install the `stablecoin-client` globally.
+
+## Run Instructions
+
+If the `stablecoin-client` was installed globally with `stack install`, then run `stablecoin-client --help`
+to see a list of the available commands.
+Otherwise, run `stack run stablecoin-client -- --help`.
```

Comment:
I'd add that `tezos-client` must be installed in order to use `stablecoin-client`.

Reply:
Done